ddl/ddl_worker.go

```diff
@@ -527,6 +539,55 @@ func (w *worker) handleDDLJobQueue(d *ddlCtx) error {
 	}
 }
 
+func getDDLJobsInQueue(t *meta.Meta, jobListKey meta.JobListKeyType) ([]*model.Job, error) {
```

Comment:
Looks that the code is copied from: https://github.com/pingcap/tidb/blob/f121cb7682894e88cfc58380924c10de99e7ed20/util/admin/admin.go#L200, right?

I think we can keep just one of them, and export it to be re-used.

Reply:
Yes. But `getDDLJobsInQueue` in `admin.go` is private. Do you think can we make it public to other package?